cmake/kokkos_enable_devices.cmake

```diff
@@ -75,30 +75,17 @@ endif()
 kokkos_device_option(OPENMPTARGET OFF DEVICE "Whether to build the OpenMP target backend")
 if(KOKKOS_ENABLE_OPENMPTARGET)
   set(ClangOpenMPFlag -fopenmp=libomp)
-  if(KOKKOS_CLANG_IS_CRAY)
-    set(ClangOpenMPFlag -fopenmp)
-  endif()
 
   compiler_specific_flags(
     Clang
     ${ClangOpenMPFlag}
     -Wno-openmp-mapping
-    IntelLLVM
-    -fiopenmp
-    -Wno-openmp-mapping
-    NVHPC
-    -mp=gpu
+    -Wno-unknown-cuda-version
+    -Wno-pass-failed
```

Comment:
Fine but this do not belong to this PR

Reply:
Yeah. Did not know if I should raise another PR for 1 line or just include it here. Will avoid next time.